### PR TITLE
Initial domains improvements

### PR DESCRIPTION
**Pull request by eadventurous**
Signed-off-by: Egor Ivkov <e.o.ivkov@gmail.com>

### CODE REVIEW

File: iroha/config.json [hunk 1]
```diff
@@ -23,5 +23,6 @@
   "PRIVATE_KEY": {
     "digest_function": "ed25519",
     "payload": "9ac47abf59b356e0bd7dcbbbb4dec080e302156a48ca907e47cb6aea1d32719e7233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0"
-  }
+  },
+  "ROOT_PUBLIC_KEY": "ed207233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0"
```
Comment (humb1t) [inline iroha/config.json:27]: maybe we can put it under some sort of configuration?
"STATE_CONFIG" or something like this?

File: iroha/src/lib.rs [hunk 2]
```diff
@@ -177,6 +153,37 @@ impl Iroha {
         }
     }
 
+    /// Returns the a map of a form domain_name -> domain, for initial domains.
+    /// `root_public_key` - the public key of a root account. Should be the same for all peers in the peer network.
+    pub fn initial_domains(root_public_key: PublicKey) -> BTreeMap<String, Domain> {
```
Comment (humb1t) [inline iroha/src/lib.rs:158]: How about `init.rs` module with this logic?

File: iroha/config.json (hunk 1 above)
Comment (eadventurous) [reply iroha/config.json:27]: If we move this logic into `init.rs` maybe it will be `INIT_CONFIG`?

Comment (humb1t) [reply iroha/config.json:27]: looks good to me - we can rename it later if needed

**Pull request merged**